get_started.sh

```diff
@@ -29,8 +29,8 @@ declare -a new_shell_config_lines=(
     # Make sure that all shells know where to find our custom gazebo models,
     # plugins, and resources. Make sure to preserve the path that already exists as well
     "export GAZEBO_MODEL_PATH=$DIR/src/sb_gazebo/models:${GAZEBO_MODEL_PATH}"\
-    "export GAZEBO_PLUGIN_PATH=/home/path/to/IGVC-2017/src/sb_gazebo/lib:${GAZEBO_PLUGIN_PATH}"\
-    "export GAZEBO_RESOURCE_PATH=/home/path/to/IGVC-2017/src/sb_gazebo/models:${GAZEBO_RESOURCE_PATH}"\
```

Comment:
Hmm. Odd these changes would register as changes, seeing as they're already on master? I'm just going to put it down to git being wierd.